### ISSUE TITLE
New install can't change RPC node.  Default RPC node no longer available

### ISSUE THREAD

**Issue by Mustard5** [aaroncox/vessel#78]
I'm pretty sure this is the problem.

I downloaded a brand new installation package (0.2.0) for windows 10.

After installing I'm attempting to add an account with owner permission (encrypted) and it seems to hang.  I opened up developer mode and I noticed an error in reaching the now defunct steem RPC node that was the default in the past.

![vesselissue](https://user-images.githubusercontent.com/378530/34961148-0f8cc73e-fa0c-11e7-931b-77574ed6f68b.PNG)

I don't have access to a 'Settings' menu, so I'm unable to switch RPC nodes.

Basically this makes a new install non-functional at this moment in time.

**Comment by aaroncox** [comment]
`0.2.6` was just released a few moments ago - please give that a shot.
